portugues/barata/Pedro/pedro.md

```diff
@@ -0,0 +1,3 @@
+Diz aí Pedro o que "cê" vai fazer. Eu vou comprar um vassoura pra me vassoura. Diz aí Pedro o que "cê" vai fazer. Eu vou comprar um vassoura pra me defender.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.